src/Sylius/Bundle/CoreBundle/Migrations/Version20230329093932.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230329093932 extends AbstractMigration
```

Comment:
As we are before the 1.13 release, we can probably prepare one combined migration for PriceHistory feature, especially since these migrations should be probably run conditionally, because of the plugin.